src/devices/third_reality.ts

```diff
@@ -91,6 +91,18 @@ const definitions: Definition[] = [
         fromZigbee: [fz.ias_water_leak_alarm_1, fz.battery],
         toZigbee: [],
         ota: ota.zigbeeOTA,
+        extend: [
```

Comment:
Could you also expose these features?

Reply:
> hi，we don't need this temporarily, it will be added later

If it's not needed, why was the cluster added?

Reply:
Sorry, I didn't explain clearly. Adding this cluster is only used on the dev console menu for the time being, and not on the expose menu.

Reply:
I see, thanks!